### ISSUE TITLE
Handling of git-remotes with names other than 'origin'

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [AltGr/opam-lock#10]
*(Accidentally posted as rgrinberg/opam-lock#4, first 😱)*

I've got a pinned dependency on another directory on my system, which is a checkout of a repository elsewhere on GitHub.

```
$ opam list
# Packages matching: installed
# Name                  # Installed # Synopsis
sedlex                  1.99.4      pinned to version 1.99.4 at git+file:///Users/ec/Sync/Code/Source/sedlex#add-4.02
```

My tooling, though, doesn't make use of an `origin` remote — each remote originating from a GitHub fork is instead named after the GitHub user who forked the project.

Unfortunately, that means that I get this error:

```
$ opam lock                  
opam-lock: internal error, uncaught exception:
           "/usr/local/bin/git remote get-url origin" exited with code 128 "fatal: No such remote 'origin'"
```

**Comment by rjbou** [comment]
Which version of opam-lock are you using?

**Comment by ELLIOTTCABLE** [comment]
```
$ opam lock --version
opam-lock: unknown option `--version'.
```

Actually, though, I really don't know how to tell? I'm not even sure which switch I installed the plugin on. (That's a surprisingly-complicated question. Is there an opam command to map plugins to switches? Should there be?)

**Edit**: Looks like it was on 0.1; is this fixed on 0.2? `:O`

**Comment by rjbou** [comment]
`opam show opam-lock` gives you all these informations.

> Is this fixed on 0.2

Yes, the error is handled in 0.2.

**Comment by AltGr** [comment]
Closing this repository, feature integrated into opam.
